docs/fundamentals/code-analysis/style-rules/ide0082.md

```diff
@@ -51,8 +51,8 @@ Dim n2 = NameOf(Int32)
 
 ## See also
 
-- ['nameof' operator](../../../csharp/language-reference/operators/nameof.md)
-- ['typeof' operator](../../../csharp/language-reference/operators/type-testing-and-cast.md#typeof-operator)
+- [nameof operator](../../../csharp/language-reference/operators/nameof.md)
+- [typeof operator](../../../csharp/language-reference/operators/type-testing-and-cast.md#typeof-operator)
```

Comment:
The rule is applicable to both C# and VB. I think the parallel doc pages for VB needs to be added too.

Reply:
Thanks @Youssef1313. This applies to almost all the IDE rules that have been documented, but all these things were just drastically increasing the scope of work for initial refactor. Additionally, adding a VB specific section to these rule docs made it quite verbose. I have used references to C# keywords/docs to keep the scope reasonable. Feel free to create follow-up PR(s) to add VB specific content to the docs - you may want to first create a PR for updating a single rule and get it reviewed from @gewarren so you are on same page on how to add this content, and then do updates to other rules.

Reply:
By the way, I don't think we should create separate pages for C# and VB, but instead add VB specific content to the same doc. IMO, maintaining 2 extremely similar doc pages for same rule, but different languages, is not scalable and docs will get out of sync and/or make it likely that future additions will missing adding VB counterpart.